app-emulation/euae/euae-0.8.29.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="E-UAE Experimental Ubiquitous Amiga Emulator"
+DESCRIPTION="This is a version of UAE, the Ubiquitous Amiga Emulator, with an emulation core largely based on WinUAE. It attempts to bring many of the great features of WinUAE to non-Windows platforms. This version now finally has a name, E-UAE, since that's what everybody was calling it anyway. The 'E' can stand for anything you fancy. Experimental, extreme, exciting, egalitarian, eggplant, . . .
+Currently it will build and run (with a varying degree of supported features) on Linux and other Unices, Mac OS X, BeOS, AmigaOS itself (either for 68k machines or PPC machines with AmigaOS 4.0 and the AmigaOS clones MorphOS and AROS). OS X requires either LibSDL or an X server for graphics output, but native graphics are supported on AmigaOS and BeOS, although, at the moment, SDL is also preferred on BeOS since the native driver is incomplete.
+E-UAE is open-source software and is made available under the terms of the GNU GPL. E-UAE is based on the work of dozens of contributors including Bernd Schmidt (the original author and maintainer of UAE), Bernie Meyer (the author of the x86 JIT compiler), Toni Wilen (the current maintainer of WinUAE), and many more.
+To make full use of E-UAE you will need access to an image of some version of the Amiga Kickstart ROM (although UAE does include a ROM emulation which may work with some old games). The Amiga Kickstart ROMs are copyrighted material and may not be freely distributed (so don't ask me for a copy). If you don't have an Amiga from which you can legally make a ROM image, a selection of ROM images are available for purchase online from Cloanto - publishers of the Amiga Forever distribution."
+HOMEPAGE="http://www.rcdrummond.net/index.html"
+COPYRIGHT="Richard Drummond"
```

Comment:
It would be nice to add the copyright years. It could be something like:
```
COPYRIGHT="2003-2016 Richard Drummond"
```
Note: I don't know if these are the correct years.